app/controllers/decidim/comparative_stats/admin/endpoints_controller.rb

```diff
@@ -35,6 +35,39 @@ def create
         def organization_endpoints
           @organization_endpoints ||= Endpoint.where(organization: current_organization)
         end
+
+        def update
+          enforce_permission_to :update, :endpoint, endpoint: current_endpoint
+
+          form = form(EndpointForm).from_params(params, endpoint: current_endpoint)
+
+          UpdateEndpoint.call(current_endpoint, form) do
+            on(:ok) do
+              flash[:notice] = I18n.t("endpoints.update.success", scope: "decidim.comparative_stats.admin")
+              redirect_to endpoints_path
+            end
+
+            on(:invalid) do
+              flash.now[:alert] = I18n.t("endpoints.update.error", scope: "decidim.comparative_stats.admin")
+              render :edit
+            end
+          end
+        end
+
+        def destroy
```

Comment:
Where is this action called?

Reply:
It's used when we delete an endpoint through the list view

Reply:
I don't see in the example you've given any use of ajax, just the same as we have now.